src/app/services/send-page-data/send-page-data.ts

```diff
@@ -0,0 +1,211 @@
+import { Injectable } from '@angular/core';
+
+import { ChangellyApiService } from '../../services/changelly-api/changelly-api';
+import { changellyConstData } from "../config";
+
+import { Observable } from 'rxjs';
+import { Subject } from 'rxjs/Subject';
+
+@Injectable()
+export class SendPageDataService {
+
+  dataStored: boolean = false
+
+  dataBundle =  {
```

Comment:
is it necessary to declare all of these like this?

could probably just do

`this.dataBundle = { errors: {} }`